assets/js/main.js

```diff
@@ -36,30 +36,50 @@ $(document).ready(function() {
     var secs_span = document.createElement("SPAN");
     secs_span.className = 'secs';
     countdown.appendChild(secs_span);
-     
+
     // update the tag with id "countdown" every 1 second
     setInterval(function () {
-     
+
         // find the amount of "seconds" between now and target
         var current_date = new Date().getTime();
         var seconds_left = (target_date - current_date) / 1000;
-     
+
         // do some time calculations
         days = parseInt(seconds_left / 86400);
         seconds_left = seconds_left % 86400;
-         
+
         hours = parseInt(seconds_left / 3600);
         seconds_left = seconds_left % 3600;
-         
+
         minutes = parseInt(seconds_left / 60);
         seconds = parseInt(seconds_left % 60);
-         
+
         // format countdown string + set tag value.
         days_span.innerHTML = '<span class="number">' + days + '</span>' + '<span class="unit">Days</span>';
         hours_span.innerHTML = '<span class="number">' + hours + '</span>' + '<span class="unit">Hrs</span>';
         minutes_span.innerHTML = '<span class="number">' + minutes + '</span>' + '<span class="unit">Mins</span>';
-        secs_span.innerHTML = '<span class="number">' + seconds + '</span>' + '<span class="unit">Secs</span>'; 
-     
+        secs_span.innerHTML = '<span class="number">' + seconds + '</span>' + '<span class="unit">Secs</span>';
+
     }, 1000);
-    
+
+	/* ======= Modal Speaker ========= */
+	$('#modal_speaker').on('show.bs.modal', function (e) {
+		var	arrayFields = ['name', 'job', 'company', 'about'],
+			arraySocial = ['twitter', 'linkedin', 'github'],
+			$link = $(e.relatedTarget);
+		$('#modal_speaker_photo').attr('src', 'assets/images/speakers/' + $link.data('photo') + '.jpg');
```

Comment:
Please use spaces instead of tabs for indentation.

Reply:
Done